include/wx/string.h

```diff
@@ -2560,6 +2570,10 @@ class WXDLLIMPEXP_BASE wxString
       if ( !ch.IsAscii() )
           m_impl.assign(wxStringOperations::EncodeNChars(n, ch));
       else
+#elif wxUSE_UNICODE_WCHAR && SIZEOF_WCHAR_T == 2
```

Comment:
Sorry, I don't understand the need for this block: isn't the case `ch >= 0x10000` already covered by `!ch.IsAscii()` above?